layouts/page/pulumi-up.html

```diff
@@ -0,0 +1,34 @@
+{{ define "main" }}
+    <div class="pulumi-up-body text-white py-12">
+        <div class="container mx-auto">
+            <section id="about">
+                <div class="neon-container">
+                    <span class="blue-neon-sign-xl">Pulumi</span>
+                    <span class="green-neon-sign-xl">UP</span>
+                </div>
+                <pulumi-date-countdown
+                    value-label-class="text-2xl"
+                    text-class="w-full text-center text-white text-5xl date-countdown"
+                    date-string="2021-04-20T09:00:00-08:00"
+                ></pulumi-date-countdown>
+                <div class="w-full flex items-center justify-center">
+                    <blockquote class="twitter-tweet invisible" data-align="center">
+                        <p lang="en" dir="ltr">We&#39;re cooking up something brand new and I really can&#39;t wait to tell y&#39;all about it. Building cloud apps will never be the same again. Stay tuned 📻</p>&mdash; J☁e Duffy (@funcOfJoe) <a href="https://twitter.com/funcOfJoe/status/1366829416429457408?ref_src=twsrc%5Etfw">March 2, 2021</a>
+                    </blockquote>
+                </div>
```

Comment:
This doesn't render in Firefox. How/where is the rendering handled?

Reply:
Firefox blocks the Twitter script from loading when you have `Enhanced Protection` on (which I believe is the default now?).

Reply:
That's right -- I'm running just with the default settings. Where is this script even located, though? I didn't see it when I poked around looking for it on Friday.

Reply:
Still not showing in Firefox, yeah.

I don't think this is a Firefox thing, though -- we have support for this in the site already, via the Hugo `{{< tweet >}}` shortcode, which works just fine in Firefox. See here, for example: https://www.pulumi.com/blog/aws-eks-managed-nodes-fargate/

I think it may just be the placement of the script tag (which may need to be positioned _after_ the element tagged `twitter-tweet`, rather than before it) or some other aspect of the way this is currently marked up. I do think we can do without the additional `script` include, though, given the way this is ultimately rendered in Hugo.